promptulate/agents/tool_agent/agent.py

```diff
@@ -24,8 +24,14 @@ def __init__(
         tools: List[Union[BaseTool, Tool]],
         llm: BaseLLM = None,
         stop_sequences: List[str] = None,
+        prefix_prompt_template: StringTemplate = StringTemplate(PREFIX_TEMPLATE),
         system_prompt_template: StringTemplate = StringTemplate(REACT_ZERO_SHOT_PROMPT),
         hooks: List[Callable] = None,
+        enable_role: bool = False,
+        profile: str = "bot",
```

Comment:
Profile and name seems similar. Would it be better to keep one of them?

Reply:
profile is a  unclear concept.

Reply:
I think so

Reply:
I think we can change "profile" to "agent_identity".

Reply:
I think we can change "profile" to "agent_identity".